server/server.js

```diff
@@ -229,4 +256,17 @@ app.get("/seller/:id/inventory", (req, res) => {
         })
 })
 
+app.post("/sellers/:id/inventory", upload.single('image'), (request, response) => {
+    const image = request.file.path;
```

Comment:
hi @ramla199 nice job on this, however now Maryam found a way where we don't need to send images as files and instead we can send just the image url, so you don't need to use `multer` and this can be a regular post request

Reply:
thanks Ellie